src/HealthGPS.Console/model_parser.cpp

```diff
@@ -67,14 +67,16 @@ load_dummy_model_definition(hgps::RiskFactorModelType type, const poco::json &op
     std::vector<hgps::core::Identifier> names;
     std::vector<double> values;
     std::vector<double> policy;
+    std::vector<int> policy_start;
     for (const auto &[key, json_params] : opt["ModelParameters"].items()) {
         names.emplace_back(key);
         values.emplace_back(json_params["Value"].get<double>());
         policy.emplace_back(json_params["Policy"].get<double>());
+        policy_start.emplace_back(json_params["PolicyStart"].get<int>());
```

Comment:
Just to confirm I get this right, you need a start value, in years since the start of the simulation, for each of the risk factors, so the policy is applied from that point. Is this correct?

Reply:
Correct. I'll clarify in the docstring.